backend/src/packages/chaiNNer_pytorch/settings.py

```diff
@@ -41,7 +48,7 @@
             if is_arm_mac
             else "Runs PyTorch in half-precision (FP16) mode for less VRAM usage. RTX GPUs also get a speedup."
         ),
-        default=False,
+        default=should_fp16 or is_arm_mac,
```

Comment:
Why is the `or is_arm_mac` here?

Reply:
Because arm Macs support fp16

Reply:
I guess I could move this over to the if condition

Reply:
Oh, okay.